wasm/src/main/scala/ir2wasm/TypeTransformer.scala

```diff
@@ -47,6 +47,17 @@ object TypeTransformer {
       case _                   => List(transformType(t))
     }
 
+  def transformTypeRef(t: IRTypes.TypeRef)(implicit ctx: ReadOnlyWasmContext): Types.WasmType =
```

Comment:
This won't be correct for `ClassRef`s of JS types. We need to use
```scala
transformType(ctx.inferTypeFromTypeRef(typeRef))
```
instead.

Note that this is not a problem in `transformType` because a `ClassType(someJSType)` is not valid in the first place. But `ClassRef(someJSType)` is valid; its corresponding type is `AnyType`.